test/e2e/specs/editor/various/block-bindings.spec.js

```diff
@@ -0,0 +1,827 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+const textCustomFieldValue = 'Value of the text_custom_field';
```

Comment:
Nit: all these `const` and `let` declarations can be moved to the function declared for `test.describe`. It could probably also be combined into a single configuration object.

Reply:
Done in [this commit](https://github.com/WordPress/gutenberg/pull/58550/commits/14079b195a8a6fd8fedef3803ac461d834392233).